### PR TITLE
branch mess up

### PR DESCRIPTION
**Pull request by ReiFan49**
and it's travis-kun is happy...

### CODE REVIEW

File: src/pages/devtools/themes/natsuiro/natsuiro.js [hunk 1]
```diff
@@ -1140,13 +1140,26 @@
...
+			
 			$(".module.activity .activity_expedition .expres_noget").hide();
 			$(".activity_expedition .expres_item").each(function(i,element){
-				var useItem = data.response["api_get_item"+(i+1)];
-				if(!!useItem && useItem.api_useitem_id > 0) {
-					gotItem = true;
-					$("img", element).attr("src", "../../../../assets/img/client/"+["bucket","ibuild","devmat","screws","","","","","","box1", "box2", "box3"][useItem.api_useitem_id]+".png");
+				var
+					useCons = data.response.api_useitem_flag[i];
+					useItem = data.response["api_get_item"+(i+1)];
```
Comment (bamboo3250) [inline src/pages/devtools/themes/natsuiro/natsuiro.js]: What if both `api_useitem_flag` and `api_get_item1` are available? One of them will be overwritten for the first `. expres_item` slot, right?

Comment (nicky008) [inline src/pages/devtools/themes/natsuiro/natsuiro.js]: For each one, right?

Comment (ReiFan49) [inline src/pages/devtools/themes/natsuiro/natsuiro.js]: `"api_useitem_flag":[4,0],"api_get_item1":{"api_useitem_id":10,"api_useitem_name":"\u5bb6\u5177\u7bb1\uff08\u5c0f\uff09","api_useitem_count":1}`
i put a comment sample data on ExpedResult listener of Natsuiro

**Comment by dragonjet** [comment]
does this fix #777?

**Pull request merged**